plugin-tester-scala/README.adoc

```diff
@@ -0,0 +1,22 @@
+= Sample repository for Gradle plugin functional testing
+
+== Run project tests
+
+[source,sh]
+----
+$ ./gradlew clean test
+----
+
+== Configure plugin version
+
+Version of the `akkaGrpc` plugin is stored in file `gradle.properties` in property named `akkaGrpcTest.pluginVersion`.
+
+This version can also be overriden from command line:
+
+[source,sh]
+----
+$ ./gradlew clean test -PakkaGrpcTest.pluginVersion=0.0.1
+----
+
+This override can be used for testing SNAPSHOT versions published to Maven local.
+So, this allow to use this sample project for local testing of published plugin and libraries.
```

Comment:
Hmmm, Id' rather we sticked to markdown. But it's not a big deal.

Reply:
After reading the contents of the file, I think it needs another pass:

 - the title seems generic
 - it'd be good to adapt the instructions to akka-grpc's use of `dynver`-based versioning.

Reply:
1. plz propose the better option, since I may be bad in creating titles :)))
1. > it'd be good to adapt the instructions to akka-grpc's use of dynver-based versioning.
that's smth i completely don't understand - can you give much more detaisl what do you mean ?